tests/Benchmarks/SystemTextJson/JsonReaderPerf.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BenchmarkDotNet.Attributes;
+using System;
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using static Benchmarks.SystemTextJson.Helper;
+
+namespace Benchmarks.SystemTextJson
+{
+    [MemoryDiagnoser]
+    public class JsonReaderPerf
+    {
+        private byte[] data;
+        private SymbolTable symbolTable;
+        private MemoryStream mem;
+        private StreamReader reader;
+
+#pragma warning disable CS0649 // Field '' is never assigned to, and will always have its default value
```

Comment:
@adamsitnik, can we do anything about these warnings that show up when we use Params?

Reply:
@ahsonkhan does it help when you make it a property?